src/trixi_timeit.jl

```diff
@@ -0,0 +1,71 @@
+# Enable debug timings `@trixi_timeit timer() "name" stuff...`.
+# This allows us to disable timings completely by executing
+# `TimerOutputs.disable_debug_timings(TrixiBase)`
+# and to enable them again by executing
+# `TimerOutputs.enable_debug_timings(TrixiBase)`
+timeit_debug_enabled() = true
+
+"""
+    disable_debug_timings()
+
+Disable all `@trixi_timeit` and `@timeit_debug` timings.
```

Comment:
```suggestion
Disable all [`@trixi_timeit`](@ref) timings.
```

We don't import `@timeit_debug` from TImerOutputs.jl.